src/main/java/uk/gov/beis/els/service/DocumentRendererService.java

```diff
@@ -53,6 +71,20 @@ public ResponseEntity processPdfApiResponse(ProcessedEnergyLabelDocument process
     return serveWithContentType(pdf, pdfRenderer.getTargetContentType());
   }
 
+  public ResponseEntity processPdfApiResponse(List<ProcessedEnergyLabelDocument> processedDocuments) {
+    List<Document> htmlDocuments = processedDocuments.stream()
+        .map(ProcessedEnergyLabelDocument::getDocument)
+        .collect(Collectors.toList());
+
+    ProcessedEnergyLabelDocument primaryDocument = processedDocuments.get(0);
+    analyticsService.sendGoogleAnalyticsEvent(primaryDocument.getClientAnalyticsToken(),
+        GoogleAnalyticsEventCategory.ENERGY_LABEL_API,
```

Comment:
Are the only differences the analytics category between these two methods? Can this not be reduced down somewhat?

Reply:
Yeah fair point, ill sort that out.

Reply:
Ah actually the response type also changes. I've updated try and remove some duplication.

![image](https://user-images.githubusercontent.com/3612419/159339492-ec940356-70b0-4c51-814a-aed53c729c55.png)

It's technically more code now 😢 , but probably a bit more readable?